src/views/Teams/Teams.test.js

```diff
@@ -0,0 +1,7 @@
+import { render } from '@testing-library/react';
+import Teams from '../../views/Teams/Teams';
+
+test('renders list of Teams', () => {
```

Comment:
This test should be async with an await findyByText to properly this the component and avoid a memory leak